src/libs/ajax/TermsOfService.ts

```diff
@@ -0,0 +1,38 @@
+import _ from 'lodash/fp';
+import { authOpts, fetchSam } from 'src/libs/ajax/ajax-common';
+
+export interface SamUserTermsOfServiceDetails {
+  latestAcceptedVersion: string;
+  acceptedOn: Date;
+  permitsSystemUsage: boolean;
+  isCurrentVersion: boolean;
+}
+
+export const TermsOfService = (signal?: AbortSignal) => {
+  return {
+    getTermsOfServiceText: async (): Promise<string> => {
+      const res = await fetchSam('termsOfService/v1/docs?doc=termsOfService', { signal });
+      return res.text();
+    },
+    getPrivacyText: async (): Promise<string> => {
+      const res = await fetchSam('termsOfService/v1/docs?doc=privacyPolicy', { signal });
+      return res.text();
+    },
+    acceptTermsOfService: async (): Promise<void> => {
+      return await fetchSam(
+        'api/termsOfService/v1/user/self/accept',
+        _.mergeAll([authOpts(), { signal, method: 'PUT' }])
+      );
+    },
+    rejectTermsOfService: async (): Promise<void> => {
```

Comment:
Do we want to make these just `accept` and `reject` since they are already namespaced under `TermsOfService`?